js/function.js

```diff
@@ -169,4 +169,24 @@ function generateRandomBlog( data ) {
 
 // CONTACT
 
-// FOOTER
\ No newline at end of file
+// FOOTER
+
+function generateSocialNetworks( data ){
+    var HTML = '',
+        position = 0,
+        icon;
+
+    if( !Array.isArray(data)){
+        return HTML;
+    }
+
+    for(; position < data.length; position++){
```

Comment:
Veikia.. Variantas.. Tačiau dažniausiai iteracijos kintamasis inicijuojamas vietoje. Funkcijos pradžioje to daryti neverta.
Vietoje "position", naudočiau "i" kintamojo pavadinimą.